src/roll20/content-script.js

```diff
@@ -741,7 +745,13 @@ async function handleOGLRenderedRoll(request) {
         atk_props["dmg2"] = convertRollToText(request.whisper, request.damage_rolls[1][1]);
         atk_props["dmg2type"] = request.damage_rolls[1][0];
     }
-
+    if (originalRequest.rollAttack && originalRequest["to-hit"] !== undefined) {
+        atk_props["mod"] = originalRequest["to-hit"];
+        if (originalRequest.character.settings["custom-roll-dice"].length > 0) {
+            atk_props["mod"] += "+" + originalRequest.character.settings["custom-roll-dice"];
```

Comment:
I think this was checking for the roll to be of type `character` : `if (originalRequest.character && originalRequest.character.type === "Character")` in the previous PR.
Also, I'd save the custom-roll-dice to a variable to avoid the long line, and also `.trim()` before checking for its length. and `(settings["custom-roll"dice"] || "").trim()` in case it's undefined.
The rest looks good to me! 👍

Reply:
@kakaroto Ah yeah, I wasn't thinking it all the way through when I pulled those checks out.

@jjchambl you want to get those things put in?

Reply:
Yes, putting it in now.

Reply:
Done: see amended PR.